src/lib/openapi/spec/feature-lifecycle-completed-schema.ts

```diff
@@ -0,0 +1,30 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const featureLifecycleCompletedSchema = {
+    $id: '#/components/schemas/featureLifecycleCompletedSchema',
+    description: 'A feature that has been marked as completed',
+    additionalProperties: false,
+    type: 'object',
+    required: ['status'],
+    properties: {
+        status: {
+            type: 'string',
+            enum: ['kept', 'discarded'],
+            example: 'kept',
+            description:
+                'The status of the feature after it has been marked as completed',
+        },
+        statusValue: {
```

Comment:
do we want to allow to send discarded with statusValue? In frontend type modelling I made this scenario impossible. But maybe openapi can be more forgiving.